### DISCUSSION TITLE
KERI Implementer's Call 06/06/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#87] [Meetings]
https://us06web.zoom.us/rec/share/NjLWhNe-ta0c3AB-t1-Mc4ZWNBBgS3a2ClAUz548X3wX6O2FvC22q9h9WH9iTrZa.IEhOnPAYtyeftL5l
Passcode: Eu*U728$

### Reports
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Please add comments ahead of tomorrow's meeting for any additional topics

**Comment by kentbull** [comment]
## SigTS
### Merged PRs
- [fix: delete contact by prefix](https://github.com/WebOfTrust/signify-ts/pull/264) - Fergal
- [Fix non-transferable prefix](https://github.com/WebOfTrust/signify-ts/pull/265) - Rodo
### New PRs
- [fix npm package warning](https://github.com/WebOfTrust/signify-ts/pull/266)

## CESRide
Nothing new the last week

**Comment by pfeairheller** [comment]
### SigPy

- No new issues or PRs

**Comment by m00sey** [comment]
keripy

- https://github.com/WebOfTrust/keripy/pull/797
- https://github.com/WebOfTrust/keripy/pull/795
- https://github.com/WebOfTrust/keripy/pull/794
- https://github.com/WebOfTrust/keripy/pull/792
- https://github.com/WebOfTrust/keripy/pull/791

**Comment by rodolfomiranda** [comment]
## KERIA
### Merged PRs
- [Add docstrings for OpenApi documentation for keria rest endpoints](https://github.com/WebOfTrust/keria/pull/252)
- [The PR attempts to catch up with KERIpy delegation logic](https://github.com/WebOfTrust/keria/pull/246)
### Pending PRs
- [Feat/delegation approval endpoint (replacement of 247 ](https://github.com/WebOfTrust/keria/pull/250)
### New issues
- [Empty REST endpoint docstrings results improper generation of OpenAPI documentation](https://github.com/WebOfTrust/keria/issues/251)
